.github/workflows/reproducible-builds.yml

```diff
@@ -0,0 +1,104 @@
+# Build Constellation CLI and check for reproducible builds
+name: Reproducible Builds
+
+on:
+  workflow_dispatch:
+  push:
```

Comment:
do we run on every push and on scheduled time? I think scheduled is enough

Reply:
Agreed